File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/HinhThucThanhToan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class HinhThucThanhToan
    {
        public string maHinhThucThanhToan { get; set; } // primary key
        public string tenHinhThucThanhToan { get; set; }

        // constructor
        public HinhThucThanhToan() { }
        public HinhThucThanhToan(HinhThucThanhToan hinhThuc)
        {
            this.maHinhThucThanhToan = hinhThuc.maHinhThucThanhToan;
            this.tenHinhThucThanhToan = hinhThuc.tenHinhThucThanhToan;
        }
        public HinhThucThanhToan(string maHinhThucThanhToan, string tenHinhThucThanhToan)
        {
            this.maHinhThucThanhToan = maHinhThucThanhToan;
            this.tenHinhThucThanhToan = tenHinhThucThanhToan;
        }
    }
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/src/mypack/HinhThucThanhToan.java
package mypack;

public class HinhThucThanhToan {

	public String maHinhThucThanhToan;	// primary key   
	public String tenHinhThucThanhToan;
    
    // constructor 
    public HinhThucThanhToan() { }
    public HinhThucThanhToan(HinhThucThanhToan hinhThuc)
    {
        this.maHinhThucThanhToan = hinhThuc.maHinhThucThanhToan;
        this.tenHinhThucThanhToan = hinhThuc.tenHinhThucThanhToan;
    }
    public HinhThucThanhToan(String maHinhThucThanhToan, String tenHinhThucThanhToan)
    {
        this.maHinhThucThanhToan = maHinhThucThanhToan;
        this.tenHinhThucThanhToan = tenHinhThucThanhToan;
    }
    
    // getter - setter
    public String getMaHinhThucThanhToan() {
		return maHinhThucThanhToan;
	}
	public void setMaHinhThucThanhToan(String maHinhThucThanhToan) {
		this.maHinhThucThanhToan = maHinhThucThanhToan;
	}
	public String getTenHinhThucThanhToan() {
		return tenHinhThucThanhToan;
	}
	public void setTenHinhThucThanhToan(String tenHinhThucThanhToan) {
		this.tenHinhThucThanhToan = tenHinhThucThanhToan;
	}
}
